appjs/views/edit_project.js

```diff
@@ -565,7 +641,13 @@ var EditProject = BaseView.extend(require('./mixins/actions'), require('./mixins
 
       var opts = {
         "schema": {
-          "title": this.model.blueprint.get('title'),
+          "title": function(){
+            if(view.model.hasPreviewType('live')){
+              return '';
+            } else {
+              return view.model.blueprint.get('title');
+            }
+          },
```

Comment:
We no longer show the title of the blueprint within the project form for blueprints that are live preview enabled.